app/helpers/application_helper.rb

```diff
@@ -1,2 +1,12 @@
+# Helper methods
 module ApplicationHelper
-end
+  def login_helper (style)
+    if current_user.is_a?(User)
+      link_to "Logout", destroy_user_session_path, method: :delete, class: style
+    else
+      (link_to "Register", new_user_registration_path, class: style) +
+      " ".html_safe +
+      (link_to "Login", new_user_session_path, class: style)	
```

Comment:
@sohilp393, Please follow the Rubocop. Run the rubocop before committing any files.
1. Trailing whitespace
2. Prefer single-quoted strings when you don't need string interpolation.
3. Use 2 (not 0) spaces for indenting an expression

Reply:
@krishnasingh001 . Removed the all rubocop warning for this file.